docs/diagrams/reassign-evolve.md

```diff
@@ -0,0 +1,43 @@
+```mermaid
```

Comment:
i'd prefer this is just in the ADR

Reply:
[dcfacd6](https://github.com/ar-io/ar-io-ant-process/pull/74/commits/dcfacd669ddaaa631f9fee1f51d32c29c6d0f2c8)